crates/node/metrics/src/recorder.rs

```diff
@@ -0,0 +1,53 @@
+//! Prometheus recorder
+
+use eyre::WrapErr;
+use metrics_exporter_prometheus::{PrometheusBuilder, PrometheusHandle};
+use metrics_util::layers::{PrefixLayer, Stack};
+use once_cell::sync::Lazy;
+
+/// The default Prometheus recorder handle. We use a global static to ensure that it is only
+/// installed once.
+pub static PROMETHEUS_RECORDER_HANDLE: Lazy<PrometheusHandle> =
+    Lazy::new(|| PrometheusRecorder::install().unwrap());
```

Comment:
imo we should still have a function for installing this instead of calling `PROMETHEUS_RECORDER_HANDLE.clone()` in the CLI